s3torchconnector/tst/unit/test_s3_client.py

```diff
@@ -82,3 +84,34 @@ def test_user_agent_always_starts_with_package_version(comments):
     if comments_str:
         assert comments_str in s3_client.user_agent_prefix
         assert comments_str in s3_client._client.user_agent_prefix
+
+
+@given(
+    part_size=integers(min_value=5 * 1024, max_value=5 * 1024 * 1024),
```

Comment:
Not blocking: Might be worth giving explicit examples for min and max as in https://hypothesis.readthedocs.io/en/latest/reproducing.html#hypothesis.example to ensure they're always tested